ui-ngx/src/app/modules/home/components/widget/config/basic/chart/flot-basic-config.component.ts

```diff
@@ -82,6 +82,8 @@ export class FlotBasicConfigComponent extends BasicWidgetConfigComponent {
       legendConfig: [configData.config.settings?.legendConfig, []],
       actions: [configData.config.actions || {}, []]
     });
+    this.flotWidgetConfigForm.get('series').valueChanges
+      .subscribe(() => this.flotWidgetConfigForm.get('datasources').updateValueAndValidity());
```

Comment:
This is an incorrect fix because needs to be fixed validation in `tb-datasources `components